### ISSUE TITLE
APN error on Android devices.

### ISSUE THREAD

**Issue by gkasireddy202** [facebook/react-native#35425]
### Description

Hi Team,

My app is getting stuck on the Samsung S22 ultra emulator and getting ANR crash reports from bugsnag.
I am running the app on the Samsung S22 emulator API 33 and android version 13.

Thanks,

### Version

React-native version :0.63.3,0.68.2

### Output of `npx react-native info`

System:
    OS: macOS 12.5.1
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 404.14 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 17.9.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.14.9 - ~/Documents/Projects/KhamSoft/Mobile/khameleonApp/node_modules/.bin/npm
    Watchman: 2022.03.21.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 23, 24, 25, 26, 27, 28, 29, 30, 31, 32
      Build Tools: 28.0.3, 29.0.2, 30.0.2, 30.0.3, 31.0.0, 32.0.0, 32.1.0, 33.0.0
      System Images: android-30 | Google APIs Intel x86 Atom, android-32 | Google APIs Intel x86 Atom_64, android-32 | Google Play Intel x86 Atom_64, android-33 | Google APIs Intel x86 Atom_64, android-33 | Google Play Intel x86 Atom_64
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.68.2 => 0.68.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

1) When I open the app on the emulator the app is struck and gets an ANR error.

### Snack, code example, screenshot, or link to a repository

<img width="1227" alt="Screenshot 2022-11-22 at 10 47 50 AM" src="https://user-images.githubusercontent.com/90603484/203228648-280b2097-468b-4288-9b54-d5e65047ad04.png">

**Comment by cortinico** [comment]
Can you provide a reproducer? Also there is no trace of React Native in the stacktrace

**Comment by gkasireddy202** [comment]
Hi,

I have tested the textinput component on the Pixel 4(Android 13)API 33 emulator and Samsung S22 ultra(Android 13) remote emulator. The keyboard is freezing and showing app is not responding pop-up on the Samsung S22 ultra emulator while typing. There are no keyboard typing issues with the Pixel 4 emulator. I have attached both scenarios videos for your reference.

Thanks,

https://user-images.githubusercontent.com/90603484/203711912-ae1aa4a1-fd5e-4cc5-8614-a5ba679728e1.mp4

https://user-images.githubusercontent.com/90603484/203712417-4968b82e-9659-42cb-92d0-ed983cf8f104.mp4
